neurodocker/utils.py

```diff
@@ -95,7 +95,7 @@ def load_json(filepath, **kwargs):
     for `json.load()`.
     """
     if filepath == '-':
-        filepath = sys.stdin.fileno()
+        return json.load(sys.stdin)
```

Comment:
Should this be `json.load(sys.stdin, **kwargs)`?

Reply:
yes, my mistake. thanks for catching